src/tests/test_utils/common_test_utils/tests/file_util_test.cpp

```diff
@@ -272,11 +272,11 @@ TEST(file_util, path_cast_unicode) {
     EXPECT_TRUE(std::u16string(u"~/狗/ǡ୫ԩϗ/にほ/ąę/ど/௸ඊƷ/狗7.txt") ==
                 std::filesystem::path(U"~/狗/ǡ୫ԩϗ/にほ/ąę/ど/௸ඊƷ/狗7.txt").generic_u16string());
 
-#if !defined(_MSC_VER) && defined(OPENVINO_CPP_VER_AT_LEAST_20)
+#if !defined(_MSC_VER) && defined(__cpp_lib_char8_t)
```

Comment:
```suggestion
defined(__cpp_lib_char8_t)
```
It should be removed